docs/UserGuide.md

```diff
@@ -194,12 +194,11 @@ Format: `delete INDEX`
 * Deletes the person at the specified `INDEX`.
 * The index refers to the index number shown in the displayed person list.
 * The index **must be a positive integer** 1, 2, 3, …​
-* Keying in multiple indexes deletes contacts at all the specified indexes of the displayed list.
+* The delete command only support deletion of **one person** at one time. 
```

Comment:
just a small nit but I think the sentence should be phrased as 
"The delete command only supports the deletion of **one person** at a time."

Reply:
Ok！Ive made the change